code/data_structures/src/tree/binary_tree/rope/rope.py

```diff
@@ -57,7 +57,9 @@ def __init__(self, text=None):
         else:
             self.head = node()
 
-    def Index(self, ind, node=self.head):
+    def Index(self, ind, node=None):
+        if node == None:
```

Comment:
Instead check if `is None`, not `== None`.

Reply:
yes I have changed it.